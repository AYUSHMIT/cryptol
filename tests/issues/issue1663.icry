:module Float
let x = 3.5 : Float32
:eval floor x
floor x
:eval ceiling x
ceiling x
:eval trunc x
trunc x
:eval roundToEven x
roundToEven x
:eval roundAway x
roundAway x
