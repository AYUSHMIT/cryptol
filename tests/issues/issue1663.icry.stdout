Loading module Cryptol
Loading module Cryptol
Loading module Float
3
3
4
4
3
3
4
4
4
4
